x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts/risk_score/index.ts

```diff
@@ -10,12 +10,13 @@ import type {
   IEsSearchRequest,
   IEsSearchResponse,
 } from '../../../../../../../../src/plugins/data/common';
+import { RISKY_HOSTS_INDEX_PREFIX } from '../../../../constants';
 import { Inspect, Maybe, TimerangeInput } from '../../../common';
 
 export interface HostsRiskScoreRequestOptions extends IEsSearchRequest {
   defaultIndex: string[];
```

Comment:
Please consider rename defaultIndex? I would confuse between indices from sourcerer.

Reply:
I agree with you that it is confusing. But it is consistent with other places. I ran a search and could find another 13 `RequestOptions` with the same property name in security_solution. It is also present on `RequestBasicOptions`, which is used in many places. So we could refactor all of them on another ticket. What do you think?